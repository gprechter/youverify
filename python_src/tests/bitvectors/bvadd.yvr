a: BV[16]
b: BV[16]
c: BV[16]

a = BV{5, 16}
b = BV{4, 16}
call c = bvadd(a, b)