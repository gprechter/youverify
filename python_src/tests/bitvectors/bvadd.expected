{
    "a": 28986,
    "b": 7,
    "c": 9,
    "d": 17,
    "e": 57972
}